sql/derived_tables/po_lines_cost.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS local.po_lines_cost;
+
+CREATE TABLE local.po_lines_cost AS 
+
+/* Create table for cost information on the purchase order line */
+
+SELECT
+	pol.id AS pol_id,
+	json_extract_path_text(pol.data, 'cost', 'listUnitPrice') AS po_lines_list_unit_price_phys,
+	json_extract_path_text(pol.data, 'cost', 'quantityPhysical') AS po_lines_quant_phys,
+	json_extract_path_text(POL.data, 'cost', 'listUnitPriceElectronic') AS po_lines_list_unit_price_elec,
+	json_extract_path_text(POL.data, 'cost', 'quantityElectronic') AS po_lines_quant_elec,
+	json_extract_path_text(POL.data, 'cost', 'additionalCost') AS po_lines_additional_cost,
+	json_extract_path_text(POL.data, 'cost', 'currency') AS po_lines_currency,
+	json_extract_path_text(POL.data, 'cost', 'discount') AS po_lines_discount,
+	json_extract_path_text(POL.data, 'cost', 'discountType') AS po_lines_discount_type,
+	json_extract_path_text(POL.data, 'cost', 'poLineEstimatedPrice') AS po_lines_estimated_price

```

Comment:
Please use lowercase for identifiers (`pol.data` rather than `POL.data`).  I will fix this after merging.